### PR TITLE
Development on work and about myself pages

### PR DESCRIPTION
**Pull request by MosDevx**
Added list of  projects completed
Added about-myself section
Added qualification and skills  section

### CODE REVIEW

File: HomePage.html [hunk 1]
```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
...
 				</section>
 	
+		</div>
+
+		
+		<div class="project-page" id="projects">
+			
+			<section >
+				<h1>My Recent <br> Works</h1>
+				<div class="underline"></div>
```
Comment (fobadara) [inline HomePage.html:76]: - [x] [REQUIRED] Kindly move the green line down as shown in your portfolio template. This is to train you to follow client requirements.

| Your Design|
|-------------|
|image|

File: HomePage.html [hunk 2]
```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
...
+			
+			<div class="about-me-page" id="about-me">
+				
+				<section class="about-me-section">
+					<h2>About Me</h2>
+					<p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+						don’t hesitate to contact me.</p>
+					<button class="resume-button">Get My Resume</button>
+			</section>
+
```
Comment (fobadara) [inline HomePage.html]: - [x] [REQUIRED] Kindly ensure that the background images present in the template are also present in yours.

|Template|
|----------|
|image|

File: HomePage.html [hunk 3]
```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
...
 					</button>
 		
 				</section>
 	
+		</div>
+
+		
+		<div class="project-page" id="projects">
+			
+			<section >
```
Comment (fobadara) [inline HomePage.html]: - [x] [REQUIRED] The section tag should be enough to wrap the about section. If you really want to wrap it with two tags then let the section be the outer tag.

File: HomePage.html [hunk 4]
```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
...
+				<h1>My Recent <br> Works</h1>
```
Comment (fobadara) [inline HomePage.html]: [REQUIRED] Kindly use another heading tag You should have only one `h1` tag in your document. This is more accessible and helpful for screen readers.

File: HomePage.html [hunk 5]
```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
 						</ul>
 
 					<button class="scroll-button">
-						<img src="Assets/Icons/scroll-button.svg" alt=""/>
+						<a href="#projects"><img src="Assets/Icons/scroll-button.svg" alt=""/></a>
 					</button>
```
Comment (fobadara) [inline HomePage.html]: [REQUIRED] Kindly ensure that your button states match the template.

|Template|
|-----------|
|image|

File: HomePage.html [hunk 6]
```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
...
+								<ul class="language-list">
+										<li>Ruby on Rails</li>
+										<li>Css</li>
+										<li>Javascript</li>
+										<li>Typescript</li>
+										<li>Html</li>
+										<li>Java</li>
+								</ul>
+							</div>
+							<button class="project-card-button"> See Project</button>
```
Comment (fobadara) [inline HomePage.html:123]: [REQUIRED] Kindly ensure that your button states match the template.

|Template|
|-----------|
|image|

Kindly follow the colour pattern for all your buttons. Just the green ones

|Template|
|---------|
|image|

File: HomePage.html [hunk 7]
```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
...
+							<h3 class="card-heading">Multi-Post Stories <br> Gain+Glory</h3>
+							<div class="card-languages">
+								<ul class="language-list">
+										<li>Ruby on Rails</li>
+										<li>Css</li>
+										<li>Javascript</li>
+										<li>Html</li>
+								</ul>
+							</div>
+							<button class="project-card-button"> See Project</button>
```
Comment (fobadara) [inline HomePage.html:145]: [REQUIRED] Kindly ensure that your button states match the template.

|Template|
|-----------|
|image|

File: HomePage.html [hunk 8]
```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
...
+								<ul class="language-list">
+										<li>Ruby on Rails</li>
+										<li>Css</li>
+										<li>Javascript</li>
+										<li>Python</li>
+			
+									
+								</ul>
+							</div>
+							<button class="project-card-button"> See Project</button>
```
Comment (fobadara) [inline HomePage.html:168]: [REQUIRED] Kindly ensure that your button states match the template.

|Template|
|-----------|
|image|

File: HomePage.html [hunk 9]
```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
...
+							<h3 class="card-heading">Multi-Post Stories <br> Gain+Glory</h3>
+							<div class="card-languages">
+								<ul class="language-list">
+										<li>Ruby on Rails</li>
+										<li>Css</li>
+										<li>Javascript</li>
+										<li>Html</li>
+								</ul>
+							</div>
+							<button class="project-card-button"> See Project</button>
```
Comment (fobadara) [inline HomePage.html:189]: [REQUIRED] Kindly ensure that your button states match the template.

|Template|
|-----------|
|image|

File: HomePage.html [hunk 10]
```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
...
+				
+			</div>
+			
+			<div class="about-me-page" id="about-me">
+				
+				<section class="about-me-section">
+					<h2>About Me</h2>
+					<p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+						don’t hesitate to contact me.</p>
+					<button class="resume-button">Get My Resume</button>
```
Comment (fobadara) [inline HomePage.html:205]: [REQUIRED] Kindly ensure that your button states match the template.

|Template|
|-----------|
|image|

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 
Hi @MosDevx, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Descriptive README
✅ PR is descriptive

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] I think you should name your feature branches in all lowercase for consistency's sake.
- [OPTIONAL] I think your feature branch name would have been good enough with work_and_aboutme
- [OPTIONAL] In your README I think it would be better to tell us how to clone your repo. Something like git clone https://github.com......


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: HomePage.html (hunk 3 above)
Comment (MosDevx) [reply HomePage.html]: @fobadara  Kindly I dont understand. the div is wrapping a whole page : with the section containing my recent works and another div containing a grid of the project cards.
What way are you suggesting?

File: HomePage.html (hunk 5 above)
Comment (MosDevx) [reply HomePage.html]: @fobadara I think this is okay because its the scroll down icon and not an actual button that will change states??

Comment (MosDevx) [reply HomePage.html]: @fobadara i've changed it to a div just in case.

File: HomePage.html (hunk 3 above)
Comment (MosDevx) [reply HomePage.html]: @fobadara Ive switched it up to make the parent a section element with two divs inside.

**Comment by MosDevx** [comment]
I'm seeing that renaming the branch will close this pull request ; so ill hold off on that; but its duly noted and I will use lowercase and better branch naming next time. Thank you.

**Comment by MosDevx** [comment]
Checks are failing because of a  lighthouse install error that i cannot solve on my end . Kindly assist.

`Run npm install -g @lhci/cli@0.7.x
npm WARN deprecated raven@2.6.4: Please upgrade to @sentry/node. See the migration guide https://bit.ly/3ybOlo7
npm WARN deprecated intl-messageformat-parser@1.8.1: We've written a new parser that's 6x faster and is backwards compatible. Please use @formatjs/icu-messageformat-parser
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
/opt/hostedtoolcache/node/12.22.12/x64/bin/lhci -> /opt/hostedtoolcache/node/12.22.12/x64/lib/node_modules/@lhci/cli/src/cli.js

> underscore@1.13.5 postinstall /opt/hostedtoolcache/node/12.22.12/x64/lib/node_modules/@lhci/cli/node_modules/underscore
> patch-package

sh: 1: patch-package: not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! underscore@1.13.5 postinstall: `patch-package`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the underscore@1.13.5 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2022-09-23T20_06_16_015Z-debug.log
Error: Process completed with exit code 1.
`

File: HomePage.html (hunk 3 above)
Comment (fobadara) [reply HomePage.html]: I was trying to explain that the section tag is enough as a divider between the different parts of your site so you don't need to wrap the section tags in a div tag

File: HomePage.html (hunk 5 above)
Comment (fobadara) [reply HomePage.html]: What I mean by states are hover, pressed and enabled.

**Comment by MosDevx** [comment]
Okay . Thanks Ive gottent it now.

We have a problem with lighthouse check. Its failing on the server on
download cause of change in the package i think.
Can we remove it form linters.yml for the sake of getting our reviews in??

**Review by cwaku — APPROVED** [review]
Hi @MosDevx 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**